Dockerfile

```diff
@@ -17,8 +17,8 @@ RUN cargo build --example empty --release && \
     rm -rf ./target/release/.fingerprint/krustlet-*
 
 # Build real binaries now
-RUN rm ./src/main.rs ./src/lib.rs
+RUN rm -f ./src/main.rs ./src/lib.rs
```

Comment:
this line can be removed entirely.

Reply:
I thought that but didn't have full context 😄.